core/src/main/java/com/oath/oak/InternalOakMap.java

```diff
@@ -627,26 +654,27 @@ OakRBuffer get(K key) {
 
         Chunk<K, V> c = findChunk(key); // find chunk matching key
         Chunk.LookUp lookUp = c.lookUp(key);
-        if (lookUp == null || lookUp.handle == null) {
+        if (lookUp == null || lookUp.valueSlice == null) {
             return null;
         }
 
-        T transformation = (T) lookUp.handle.transform(transformer);
-        return transformation;
+        AbstractMap.SimpleEntry<ValueUtils.ValueResult, T> res = ValueUtils.transform(lookUp.valueSlice, transformer);
```

Comment:
First, require a comment of what is going on here.
Second, when ValueUtils.transform may return RETRY? When it was deleted? If so why not to return NULL as it is deleted? Why to retry?

Reply:
I added javadocs to ValueUtils.
When `RETRY` is returned it means that the value was moved, not deleted.